packages/theme/src/cli/commands/theme/dev.ts

```diff
@@ -110,6 +111,11 @@ You can run this command only in a directory that matches the [default Shopify t
       env: 'SHOPIFY_FLAG_OPEN',
       default: false,
     }),
+    'dev-preview': Flags.boolean({
+      hidden: true,
+      description: 'Enables the develoepr preview of the typescript `theme dev` implementation.',
```

Comment:
I personally believe we can omit the TypeScript detail from users, but it's a minor detail.